data/assets/scene/solarsystem/missions/juice/transforms.asset

```diff
@@ -0,0 +1,76 @@
+local jupiterTransforms = asset.require('scene/solarsystem/planets/jupiter/transforms')
+local kernels = asset.require('./kernels')
+
+
+local Juice = {
+  Identifier = "Juice",
+  Parent = jupiterTransforms.JupiterBarycenter.Identifier,
+  Transform = {
+    Translation = {
+      Type = "SpiceTranslation",
+      Target = "JUICE",
+      Observer = "JUPITER BARYCENTER",
+      Kernels = kernels.Kernels
+    },
+    Rotation = {
+      Type = "SpiceRotation",
+      SourceFrame = "JUICE_SPACECRAFT",
+      DestinationFrame = "GALACTIC",
+      Kernels = kernels.Kernels
+    },
+  },
+  GUI = {
+    Name = "Juice",
+    Path = "/Solar System/Missions/Juice"
+  }
+}
+
+local GanymedeBarycenter = {
+  Identifier = "GanymedeBarycenter",
+  Parent = jupiterTransforms.JupiterBarycenter.Identifier,
+  Transform = {
+    Translation = {
+      Type = "SpiceTranslation",
+      Target = "GANYMEDE",
+      Observer = "JUPITER BARYCENTER",
+      Kernels = kernels.Kernels
+    }
+  },
+  GUI = {
+    Path = "/Solar System/Planets/Jupiter/Moons"
+  }
+}
+
+local GPHIO = {
+  Identifier = "GPHIO",
+  Parent = GanymedeBarycenter.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "SpiceRotation",
+      SourceFrame = "GPHIO",
+      DestinationFrame = "GALACTIC",
+      Kernels = kernels.Kernels
+    }
+  },
+  GUI = {
+    Name = "GPHIO",
```

Comment:
What is this? Would be nice with a short description for SGN (and all others, really). Would make the profile more usable and understandable for our users